apps/throttling.py

```diff
@@ -0,0 +1,76 @@
+import logging
+from calendar import monthrange
+from datetime import datetime
+from influxdb_metrics.loader import log_metric
+
+from rest_framework import throttling
+
+LOG = logging.getLogger('transmission')
+
+
+class MonthlyRateThrottle(throttling.SimpleRateThrottle):
+    def __init__(self):
+        month = datetime.now().month
```

Comment:
I'd probably rather have us call datetime.now() once, and then just pass `now.year, now.month` into `monthrange`

Reply:
makes sense. Should I also set self.now then too?

Reply:
No, self.now needs to be set at request-time